salt/modules/xml.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+'''
+XML file mangler
```

Comment:
mangler or manager? Is it supposed to rip, shred, and tear xml files (Who doesn't want that? :trollface: )

Reply:
It was a working title :) Adjusted!